### DISCUSSION TITLE
Load fightfx from character def file

### DISCUSSION THREAD

**Discussion by zzzz0424** [ikemen-engine/Ikemen-GO#1441] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

As the title suggests, this function introduces fightfx.sff, fightfx.air, and common.snd, which are originally loaded from fight.def, into the character's folder and loads them from the character's def file.

### Describe the solution you'd like

It is a little similar to stcommon in the [Files] column of the character, which loads fight.def if the description of loading files is omitted or the files do not exist, and uses the character's files if they do exist.

Although rare, there are some characters in MUGEN that cannot display effects properly without replacing this part (e.g., [toma's marisa](https://web.archive.org/web/20070311095117/http://page.freett.com/gate0753cvt24/toma_marisa.html) and [FightFx Pack](https://web.archive.org/web/20070318004435/http://saihate-house0154-06.hp.infoseek.co.jp/data/VerEffect.dgc)).
However, to solve this problem, it is necessary to replace the lightfx, and this can spoil the effect of some of the lifebars.
To solve this problem, we would like to register fightfx.sff, fightfx.air, and common.snd in the [Files] field for some characters.

I know that IKEMEN has already added the Fx function in fight.def, but I thought it would be more convenient if this function could be added to the character as well, so I sent a request message.

Example description:
[Files]
cmd = kfm.cmd
cns = kfm.cns 
st = kfm.cns
stcommon = common1.cns
sprite = kfm.sff
anim = kfm.air
sound = kfm.snd
ai = kfm.ai 
movelist = movelist.dat 
fxsprite = fightfx.sff
fxanim = fightfx.air
fxsnd = common.snd

**Comment by NeatUnsou** [comment, marked as answer]
implement by #2497
